ControlScheme.json

```diff
@@ -55,6 +55,55 @@
     },
     "Left Bumper (Xbox)"
   ],
+  "holdBothClimbersPosition": [
```

Comment:
Why use a separate button to hold and a separate button to retract? The old system of just 1 button that extends when held, retracts when let go, and holds when finished works pretty well and is much less complicated